### PR TITLE
Geospatial Analytics Access token use

### PR DESCRIPTION
**Pull request by elljoh**
#### Added
* a page for Geospatial Analytics API authentication.

#### Changed
* Main table of contents to reference the new page; and
* Move the existing API token how to under applicable heading
* Update authorization server Uri

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -16,19 +16,22 @@ This github repository contains documentation and API usage examples that can be
...
+#### API Authentication
+
+* [How to generate and use access tokens](./geospatial-api.md#authentication)
+
+#### Custom Geospatial Queries
 
 * [Performing Custom Geospatial Queries, Registration and Visualization](./custom-geospatial-query-extension.md)
 
 ### Agriculture Add-on Specific Examples and Samples
+* [How to generate auth tokens](./api-tokens.md)
```
Comment (elljoh) [inline docs/README.md:34]: The auth token how to might be a bit hidden now.

Perhaps a more general TOC heading might be better with each component listed.

**Comment by elljoh** [comment]
@sushain-pandit @marchmallow when either of you have a chance to review this, much appreciated.

/John

File: docs/custom-geospatial-query-extension.md [hunk 2]
```diff
@@ -1,124 +1,119 @@
...
-      isInterpolated: true,
-      extendMinimumColor: false,
-      extendMaximumColor: true,
-      invalidDataValue: -9999
-    }
-  }}'`
-
-
-<b>Example response</b>: 
+    id: 'customQuery-staging-test01',
```
Comment (elljoh) [inline docs/custom-geospatial-query-extension.md:78]: This `PUT` data payload needs further refinement from the look of the JSON and quoting for usage in shell commands.

Comment (elljoh) [reply docs/custom-geospatial-query-extension.md]: Updated the JSON payload syntax to at least pass when provided to `jq -r .`

**Review by sushain-pandit — APPROVED** [review]

**Pull request merged**